pom.xml

```diff
@@ -40,7 +40,8 @@
     <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
 
     <!-- default bytecode version for src/main -->
-    <main.java.version>1.7</main.java.version>
+    <!-- IntelliJ isn't clever enough to have different language levels for tests vs. main code -->
+    <main.java.version>1.8</main.java.version>
```

Comment:
yeah, I'm sorry, I'd been good about keeping this out of the commit (I know it's not going in), but an errant `git commit -a` seems to have undone that effort.

Thanks for the JIRA and the workaround!